src/components/action-popover/action-popover-menu/action-popover-menu.component.tsx

```diff
@@ -19,7 +26,7 @@ export interface ActionPopoverMenuBaseProps {
   /** A unique ID for the menu */
   menuID?: string;
   /** Callback to set the index of the focused item */
-  setFocusIndex?: (args: number) => void;
+  setFocusIndex?: (args?: number) => void;
```

Comment:
As I said in the item file comment:

I'll double check this. I'm pretty sure I was getting the TS error because of the `firstMatch` and `nextMatch` could potentially be `undefined` in the action popover menu keyboard nav function.